cgmanifest.json

```
@@ -0,0 +1,14 @@
+{
+    "Registrations": [
+        {
```

Verify the JSON is formatted correctly (ignore this feedback if it is already correct).
I would have thought this would be like: `[{`

```suggestion
    "Registrations": [{
```

---

VS auto-format suggestion

```suggestion
{
  "Registrations": [
    {
      "Component": {
        "Type": "other",
        "Other": {
          "Name": "cmake module for adding code coverage",
          "Version": "any version",
          "DownloadUrl": "https://github.com/bilke/cmake-modules/blob/master/CodeCoverage.cmake"
        }
      }
    }
  ]
}
```

---

it is like that, the `{` is on the next line

---

https://docs.opensource.microsoft.com/tools/cg/cgmanifest.html